src/handlers/oidc.rs

```diff
@@ -115,21 +117,33 @@ broker_handler!(Auth, |app, req| {
     }
 
     if app.providers.contains_key(&email_addr.domain.to_lowercase()) {
-
         // OIDC authentication. Redirect to the identity provider.
         let auth_url = oidc_bridge::request(app, email_addr, client_id, nonce, &parsed_redirect_uri)?;
         Ok(Response::with((status::SeeOther, modifiers::Header(Location(auth_url.to_string())))))
 
     } else {
+        let langtag_de: LanguageTag = "de".parse().unwrap();
+        let accept_language = req.headers.get::<AcceptLanguage>().expect("No accepted language set");
+        let lang_file = if langtag_de.matches(&accept_language[0].item) {
+            "lang/german.mo"
+        } else {
+            "lang/english.mo" // fallback, for easier catalog creation below
+        };
```

Comment:
Hm. We could probably try to select a translation file based on the langtag-language?